tools/docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
-RUN apk add cargo && wget https://github.com/mbrubeck/agate/archive/refs/tags/v3.0.2.tar.gz && tar xzf v3.0.2.tar.gz && mv /agate-3.0.2 /agate && cd agate && cargo build --release
+RUN apk add cargo jq && wget -O source.tar.gz https://github.com/mbrubeck/agate/archive/refs/tags/$(wget -qO- https://api.github.com/repos/mbrubeck/agate/releases/latest | jq -r ".name").tar.gz && tar xzf source.tar.gz && mv /agate-* /agate && cd agate && cargo build --release
```

Comment:
Probably better to use `tarball_url` from that API response.

Also you can avoid adding jq by using sed:
```bash
wget -qO- https://api.github.com/repos/mbrubeck/agate/releases/latest | sed -nE 's/^.*"tarball_url"\s*:\s*"([^"]+)".*$/\1/p'
```

Reply:
I'll have to trust you with this sed command...

Reply:
😅 it should be right